analysis/model/make_model_input.R

```diff
@@ -59,6 +59,38 @@ for (i in 1:nrow(active_analyses)) {
   # Restrict to required variables -----------------------------------------------
   print('Restrict to required variables')
   
+  # For models with strata = NULL
+  if (active_analyses$name[i] == "cohort_vax-sub_covid_hospitalised-parkinson_disease") {
```

Comment:
This exception is for a specific name, which we want to avoid. You can either make the if statement 'for models with strata = NULL' (as suggested in your comment) OR, better yet, create a vector with all variables and remove NULL then call `input <- input[,VECTOR]`. The second option is preferable as this works if covariates or other things end up as NULL in the future.